polaris-react/locales/en.json

```diff
@@ -176,7 +176,7 @@
       "selectAllLabel": "Select all {resourceNamePlural}",
       "selected": "{selectedItemsCount} selected",
       "undo": "Undo",
-      "selectAllItems": "Select all {itemsLength}+ {resourceNamePlural}",
+      "selectAllItems": "Select all {resourceNamePlural} in this store",
```

Comment:
Just curious to know if we've considered the case where the table is actually filtered (like in orders). 

Couldn't the new copy be a bit misleading since we wouldn't be selecting all orders in the store, but al orders in the store matching the existing search?

Reply:
Not my area of expertise, but a quick look at the code shows that [we extract the query when the selection is all](https://github.com/Shopify/web/blob/main/app/sections/Orders/OrderListCustomizable/OrderListCustomizable.tsx#L1453), which matches my expectations on how the feature should work.

cc @AndrewMusgrave, @sethomas, just in case